live-site/components/event-schedule-section/event-item/EventItem.tsx

```diff
@@ -31,15 +43,31 @@ const EventItem: React.FC<EventItemProps> = ({ eventItem }) => {
             <StyledEventItemHeader>{eventItem.eventName}</StyledEventItemHeader>
             <StyledPinpointLocationContainer>
               <StyledPinpoint src={pinpoint} />
-              <StyledEventitemLocation>
+              <StyledEventItemLocation>
                 {eventItem.eventLocation}
-              </StyledEventitemLocation>
+              </StyledEventItemLocation>
             </StyledPinpointLocationContainer>
           </StyledEventItemNameLocationContainer>
         </StyledEventItemLeftSideContainer>
-          <StyledArrow src={arrow} />
-      </StyledEventItemMobileContainer>
-    </StyledEventItemContainer>
+        {arrowPresent && (
+          <StyledArrow
+            src={arrow}
+            onClick={(): void => openEventItem(eventItem.id)}
+            isOpen={isOpen}
+          />
+        )}
+        <StyledTagsContainer>
+        {eventItem.tags.map( (tag) => (<EventItemTag tagType={tag} />))}
```

Comment:
oh also include a `key` prop here so react is happy https://beta.reactjs.org/learn/rendering-lists#keeping-list-items-in-order-with-key